packages/world/src/interfaces/IModule.sol

```diff
@@ -10,7 +10,7 @@ bytes4 constant MODULE_INTERFACE_ID = IModule.getName.selector ^
   ERC165_INTERFACE_ID;
 
 interface IModule is IERC165 {
-  error RequiredModuleNotFound(string resourceSelector);
+  error RequiredModuleNotFound(string moduleIdString);
   error RootInstallModeNotSupported();
   error NonRootInstallNotSupported();
```

Comment:
maybe for followup: should we make these more consistent by removing "mode" from one or adding it to the other?

Reply:
Oh I mean "root install _mode_ not supported" vs "non root install not supported", we call it a "mode" in one instance but not the other

but yeah, can resolve as part of naming pass in #1126

Reply:
> Oh I mean "root install mode not supported" vs "non root install not supported", we call it a "mode" in one instance but not the other

I understood, I meant we can do this renaming as part of the naming pass in 1126